atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -451,10 +451,8 @@ def add_log_type(output):
 
         # Read config from the registry using v2 schema 2 digest
         registries = self.workflow.push_conf.docker_registries
-        if registries:
-            config = copy.deepcopy(registries[0].config)
-        else:
-            config = {}
+        assert registries, RuntimeError("registry config is required")
```

Comment:
Is this meant to raise a `RuntimeError`? I just tried this interactively, and it seems that a failed assertion will always result in an `AssertionError`, like so:
```python
>>> assert False, RuntimeError('error message')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: error message

Reply:
Fixed.

Reply:
Side note: never use asserts instead of ifs in code. Asserts are just for development to make sure that codepaths don't change unexpectedly. Python has option to disable asserting, it will be never executed with `python -O`.

Reply:
Removed it.